### PR TITLE
chore(demo): migrate documentation page `BadgeNotification` to standalone components

### PR DESCRIPTION
**Pull request by KrollikRoddzer**
Partially solves https://github.com/taiga-family/taiga-ui/issues/6957
Done: Migrated `Badge Notification` example components to standalone.

### CODE REVIEW

File: projects/demo/src/modules/components/badge-notification/badge-notification.component.ts [hunk 1]
```diff
@@ -1,14 +1,40 @@
 import {Component} from '@angular/core';
 import {changeDetection} from '@demo/emulate/change-detection';
-import type {TuiRawLoaderContent} from '@taiga-ui/addon-doc';
+import {TuiExamplePipe} from '@demo/utils';
+import {
+    TuiDocCodeModule,
+    TuiDocDemoModule,
+    TuiDocDocumentationModule,
+    TuiDocExampleModule,
+    TuiDocPageModule,
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/badge-notification/badge-notification.component.ts:9]: `TuiAddonDocModule` ?

File: projects/demo/src/modules/components/badge-notification/examples/2/index.ts [hunk 2]
```diff
@@ -1,9 +1,13 @@
 import {Component} from '@angular/core';
 import {changeDetection} from '@demo/emulate/change-detection';
 import {encapsulation} from '@demo/emulate/encapsulation';
+import {TuiPlatformModule} from '@taiga-ui/cdk';
+import {TuiBadgeNotificationComponent} from '@taiga-ui/kit';
 
 @Component({
+    standalone: true,
     selector: 'tui-badge-notification-example-2',
+    imports: [TuiPlatformModule, TuiBadgeNotificationComponent],
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/badge-notification/examples/2/index.ts:10]: Is `TuiPlatformModule` actually required for this example ?

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/components/badge-notification/examples/2/index.ts (hunk 2 above)
Comment (KrollikRoddzer) [reply projects/demo/src/modules/components/badge-notification/examples/2/index.ts:10]: Yes, we are using tuiPlatform in html template

Comment (nsbarsukov) [reply projects/demo/src/modules/components/badge-notification/examples/2/index.ts:10]: platform

Comment (nsbarsukov) [reply projects/demo/src/modules/components/badge-notification/examples/2/index.ts:10]: https://github.com/taiga-family/taiga-ui/blob/b36689acb658e3c2ebef88182f90b03b98ef00fc/projects/demo/src/modules/components/badge-notification/examples/2/index.html#L1-L4

File: projects/demo/src/modules/components/badge-notification/badge-notification.component.ts [hunk 3]
```diff
@@ -1,14 +1,29 @@
...
+import {TuiBadgeNotificationExample1} from './examples/1';
+import {TuiBadgeNotificationExample2} from './examples/2';
 
 @Component({
+    standalone: true,
     selector: 'example-badge-notification',
+    imports: [
+        TuiBadgeNotificationComponent,
+        TuiExamplePipe,
+
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/badge-notification/badge-notification.component.ts:17]: Empty line ?

File: projects/demo/src/modules/components/badge-notification/badge-notification.component.ts [hunk 4]
```diff
@@ -1,14 +1,29 @@
...
+        TuiAddonDocModule,
+
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/badge-notification/badge-notification.component.ts:19]: Empty line ?

**Review by nsbarsukov — CHANGES_REQUESTED** [review]

**Review by nsbarsukov — APPROVED** [review]

**Review by splincode — APPROVED** [review]

**Pull request merged**